integration-cli/docker_cli_external_volume_driver_test.go

```diff
@@ -1,5 +1,3 @@
-// +build !windows
```

Comment:
Looks like you both moved this one (`_unix_test` -> `_test`) and removed the `!windows` build-tag ?